src/main/java/com/emc/mongoose/base/load/step/client/LoadStepClientBase.java

```diff
@@ -118,9 +119,26 @@ private static void initFileManagers(final List<String> nodeAddrs, final List<Fi
 		// local file manager
 		fileMgrsDst.add(FileManager.INSTANCE);
 		// remote file managers
-		nodeAddrs.stream().map(FileManagerClient::resolve).forEachOrdered(fileMgrsDst::add);
+		nodeAddrs.stream().map(nodeAddr -> resolveFileManagerWithRetries(nodeAddr, 7)).forEachOrdered(fileMgrsDst::add);
 	}
 
+	private static FileManagerService resolveFileManagerWithRetries(final String nodeAddrWithPort, final int maxRetries) {
+		FileManagerService fms = null;
+		int retryCount = 0;
+		while (null == fms && retryCount < maxRetries) {
+			fms = FileManagerClient.resolve(nodeAddrWithPort);
+			retryCount++;
+			try {
+				Thread.sleep(1000L * (int) Math.pow(2, maxRetries - retryCount));
```

Comment:
Is the backoff expected to be reversal? For example:
```
2^7
2^6
...
2
```
Also, I think it could be useful to set a limit to maximal delay, thus in case `maxRetries` would be accidentally set to any high number, a retry wouldn't take an extraordinary long time.

Reply:
yep, you are right, i've reversed the loop for count to go up, but forgot to change the sleep time. for some reason it didn't show up in the manual test.

Reply:
As for the limit - idk how expected that is according to the concept, but I can agree that it makes 100% sense